src/webui/abstractwebapplication.cpp

```diff
@@ -187,42 +197,13 @@ bool AbstractWebApplication::sessionInitialize()
     return false;
 }
 
-bool AbstractWebApplication::readFile(const QString& path, QByteArray &data, QString &type)
+QString AbstractWebApplication::getContentType(const QString &path)
 {
     QString ext = "";
```

Comment:
did you missed to removed this redundant assignment?

Reply:
Oops... I recall I have it done... Apparently there are two similar places with this code was later.